javascript/packages/orchestrator/src/orchestrator.ts

```diff
@@ -219,13 +221,17 @@ export async function start(
       },
     );
 
-    // create namespace
-    await client.createNamespace();
+    // create namespace if ZOMBIE_NAMESPACE is not set
+    if (!process.env.ZOMBIE_NAMESPACE) {
+      await client.createNamespace();
+    }
 
     // setup cleaner
     if (!opts.monitor) {
-      cronInterval = await client.setupCleaner();
-      debug("Cleaner job configured");
+      if (!process.env.ZOMBIE_CLEANER_DISABLED) {
```

Comment:
Why we need this extra `env` var? the `monitor` options is set to `false` by default.

Reply:
When I run it locally it still tried to create the cronjob